regexes.yaml

```diff
@@ -10,6 +10,9 @@ user_agent_parsers:
   - regex: '(SoundCloud-Android)/(\d+)\.(\d+)\.(\d+)'
     family_replacement: 'SoundCloud'
 
+  - regex: '(Android) (\d+)\.(\d+)\.(\d+)'
```

Comment:
@dariuskilstein should this now be lower, with the other android section? Is this regex in fact covered by the ones below, e.g. `'Android[\- ][\d]+\.[\d]+\.[\d]+; ([A-Za-z0-9 _-]+) '`?